tests/storyscript/hub/sdk/JsonFixtureHelper.py

```diff
@@ -0,0 +1,25 @@
+import json
+import os
+
+
+class JsonFixtureHelper:
+    """
+    The JsonFixtureHelper just makes it easier to load fixture data from the json_fixture.json file. It's a massive
+    confusing blob file with some pre-loaded fixture data.
```

Comment:
```suggestion
    blob file with some pre-loaded fixture data.
```

We shouldn't say this in our official SDK.